src/test/java/org/alliancegenome/curation_api/VariantFmsITCase.java

```diff
@@ -0,0 +1,218 @@
+			body("entity.sourceGeneralConsequence.curie", is("SO:0001578")).
+			body("entity.curatedVariantGenomicLocations", hasSize(1)).
+			body("entity.curatedVariantGenomicLocations[0].hgvs", is("NC_003279.8:g.1A>T")).
+			body("entity.curatedVariantGenomicLocations[0].relation.name", is("located_on")).
+			body("entity.curatedVariantGenomicLocations[0].variantGenomicLocationAssociationObject.name", is("I")).
+			body("entity.curatedVariantGenomicLocations[0].start", is(1)).
+			body("entity.curatedVariantGenomicLocations[0].end", is(1000)).
+			body("entity.alleleVariantAssociations", hasSize(2)).
+			body("entity.alleleVariantAssociations[0].relation.name", is("has_variant")).
+			body("entity.alleleVariantAssociations[0].alleleAssociationSubject.modEntityId", is("WB:AlleleWithVar1")).
+			body("entity.alleleVariantAssociations[1].relation.name", is("has_variant")).
+			body("entity.alleleVariantAssociations[1].alleleAssociationSubject.modEntityId", is("WB:AlleleWithVar2")).
+			body("entity.relatedNotes", hasSize(1)).
+			body("entity.relatedNotes[0].internal", is(false)).
+			body("entity.relatedNotes[0].freeText", is("This is an updated test note.")).
+			body("entity.relatedNotes[0].noteType.name", is("comment")).
+			body("entity.relatedNotes[0].references[0].curie", is(reference2)).
+			body("entity.crossReferences", hasSize(1)).
+			body("entity.crossReferences[0].referencedCurie", is("TEST:WBVar00252637")).
+			body("entity.crossReferences[0].displayName", is("TEST:WBVar00252637")).
+			body("entity.crossReferences[0].resourceDescriptorPage.name", is("homepage"));
+
+	}
+	
+	@Test
+	@Order(3)
+	public void variantFmsBulkUploadMissingRequiredFields() throws Exception {
+		HashMap<String, HashMap<String, Integer>> params = new HashMap<>();
+		params.put("Entities", createCountParams(1, 1, 0, 0));
+		params.put("Locations", createCountParams(1, 1, 0, 0));
+		params.put("Associations", createCountParams(1, 1, 0, 0));
+		
+		checkBulkLoadRecordCounts(variantFmsBulkPostEndpoint, variantFmsTestFilePath + "MR_01_no_start.json", params);
+		checkBulkLoadRecordCounts(variantFmsBulkPostEndpoint, variantFmsTestFilePath + "MR_02_no_end.json", params);
+		checkBulkLoadRecordCounts(variantFmsBulkPostEndpoint, variantFmsTestFilePath + "MR_03_no_sequence_of_reference_accession_number.json", params);
+		checkBulkLoadRecordCounts(variantFmsBulkPostEndpoint, variantFmsTestFilePath + "MR_04_no_type.json", params);
+		checkBulkLoadRecordCounts(variantFmsBulkPostEndpoint, variantFmsTestFilePath + "MR_05_no_genomic_reference_sequence.json", params);
+		checkBulkLoadRecordCounts(variantFmsBulkPostEndpoint, variantFmsTestFilePath + "MR_06_no_genomic_variant_sequence.json", params);
+
+		params.put("Entities", createCountParams(1, 0, 1, 0));
+		params.put("Locations", createCountParams(1, 0, 1, 0));
+		checkBulkLoadRecordCounts(variantFmsBulkPostEndpoint, variantFmsTestFilePath + "MR_07_no_allele_id.json", params);	
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.regexp.RegexpSinglelineJavaCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line must not have trailing whitespace
<!-- __reviewdog__:ChAyMzE4ZTg1NjVlYzAwNzM2EgpjaGVja3N0eWxl -->